### PR TITLE
Bash for bsd

### PR DESCRIPTION
**Pull request by raphael-proust**
Starting to work on https://github.com/ocaml/opam-repository/issues/22457

### CODE REVIEW

File: packages/conf-bash/conf-bash.1/opam [hunk 1]
```diff
@@ -0,0 +1,13 @@
+opam-version: "2.0"
+maintainer: "code@bnwr.net"
+homepage: "https://www.gnu.org/software/bash/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-3.0-or-later"
+build: ["bash" "--version"]
+depexts: [
+  ["bash"] {os-family = "bsd"}
```
Comment (avsm) [inline packages/conf-bash/conf-bash.1/opam:8]: Also isn't zsh default on homebrew now, so bash would be needed there?

Comment (mseri) [reply packages/conf-bash/conf-bash.1/opam:8]: For the moment macos keeps shipping bash though
```
$ /bin/bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)
Copyright (C) 2007 Free Software Foundation, Inc.
```
I am not sure I would like an unnecessary extra install of bash in that case

Comment (raphael-proust) [reply packages/conf-bash/conf-bash.1/opam:8]: @avsm the aim is to allow packages that call `bash` explicitly to specify that system dependency.

Comment (avsm) [reply packages/conf-bash/conf-bash.1/opam:8]: Fine by me.

File: packages/oranger/oranger.2.0.1/opam [hunk 2]
```diff
@@ -21,6 +21,7 @@ depends: [
   "minicli" {with-test & >= "5.0.0"}
   "cpm" {with-test}
   "conf-gnuplot" {with-test}
+  "conf-bash" {build}
```
Comment (kit-ty-kate) [inline packages/oranger/oranger.2.0.1/opam:24]: is this package really using bash? I can only see `sh` used here

Comment (raphael-proust) [reply packages/oranger/oranger.2.0.1/opam:24]: The file `packages/oranger/oranger.2.0.1/files/compile_ranger.sh` has a `#!/bin/bash` shebang. And in the repository the file `install_ranger.sh` also has the bash shebang.

Comment (kit-ty-kate) [reply packages/oranger/oranger.2.0.1/opam:24]: `compile_ranger.sh` is not used anywhere as far as i can see, and `install_ranger.sh` is called directly using `sh` which bypasser the shebang

Comment (raphael-proust) [reply packages/oranger/oranger.2.0.1/opam]: I removed this for this package and the other `oranger` packages.

**Pull request merged**